A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml

```diff
@@ -520,7 +514,7 @@
          <CheckpointDesc SubjectTT="TT:T/O DATA: Flaps Lever" ExpectationTT="TT:CHECK" />
       </Checkpoint>
       <Checkpoint Id="A320_TAXI_TO_DATA_VSPEEDS">
-         <CheckpointDesc SubjectTT="TT:T/O DATA: V1, Vr, V2 (INOP)" ExpectationTT="TT:REINSERT" />
+         <CheckpointDesc SubjectTT="TT:T/O DATA: V1, VR, V2 (INOP)" ExpectationTT="TT:REINSERT" />

```

Comment:
I'm not sure about this. In Documentation V Rotate is almost always referred to as Vr and not VR.

Reply:
Ok, thanks for the feedback, I now changed this back and the other (existing) occurrence of VR to Vr